From 9e8af3b99c214afce6102c1e944c79b3f6e6aa39 Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Tue, 27 Feb 2018 18:38:16 +0000
Subject: [PATCH 357/420] {chiron,sagit}_defconfig: disable PAN emulation

 * This is a workaround to a bad implementation around
   the Elliptic Labs' proximity sensor.

Change-Id: Iea8e3314d1cda85249145b5361b5b5db00877d9e
---
 arch/arm64/configs/chiron_defconfig | 1 -
 arch/arm64/configs/sagit_defconfig  | 1 -
 2 files changed, 2 deletions(-)

diff --git a/arch/arm64/configs/chiron_defconfig b/arch/arm64/configs/chiron_defconfig
index 56ee063d1637..4349c0e6982c 100644
--- a/arch/arm64/configs/chiron_defconfig
+++ b/arch/arm64/configs/chiron_defconfig
@@ -65,7 +65,6 @@ CONFIG_ARMV8_DEPRECATED=y
 CONFIG_SWP_EMULATION=y
 CONFIG_CP15_BARRIER_EMULATION=y
 CONFIG_SETEND_EMULATION=y
-CONFIG_ARM64_SW_TTBR0_PAN=y
 CONFIG_RANDOMIZE_BASE=y
 # CONFIG_RANDOMIZE_MODULE_REGION_FULL is not set
 # CONFIG_EFI is not set
diff --git a/arch/arm64/configs/sagit_defconfig b/arch/arm64/configs/sagit_defconfig
index 56ee063d1637..4349c0e6982c 100644
--- a/arch/arm64/configs/sagit_defconfig
+++ b/arch/arm64/configs/sagit_defconfig
@@ -65,7 +65,6 @@ CONFIG_ARMV8_DEPRECATED=y
 CONFIG_SWP_EMULATION=y
 CONFIG_CP15_BARRIER_EMULATION=y
 CONFIG_SETEND_EMULATION=y
-CONFIG_ARM64_SW_TTBR0_PAN=y
 CONFIG_RANDOMIZE_BASE=y
 # CONFIG_RANDOMIZE_MODULE_REGION_FULL is not set
 # CONFIG_EFI is not set
-- 
2.15.0

